src/wwwroot/app/bot-config.js

```diff
@@ -111,6 +122,11 @@ export default function _config(
          controller: thesaurusesController
       });
 
+   $locationProvider.html5Mode({
```

Comment:
Html5 mode transforms angularjs routes of _example.com/#!/home_ to  _example.com/home_